sc-memory/sc-memory/sc_scs_helper.cpp

```diff
@@ -95,9 +100,27 @@ class StructGenerator
     }
   }
 
-  void SetSCsGlobalIdtf(std::string const & idtf, ScAddr const & addr)
+  ScAddrVector SetSystemIdtf(std::string const & idtf, ScAddr const & addr)
```

Comment:
After this change, we will have two ways of system idtf generation: this and m_ctx->HelperSetSystemIdtf. As result, we should support this two ways.

I think, we should leave only one way